File: Modul 3 Sem 1/C/PRAK304-2410817110005-MuhammadRizqanKarim.c
#include <stdio.h> 
 
int main() { 
    int bil; 
    printf("Masukkan bilangan (0-99): "); 
    scanf("%d", &bil); 
 
    if (bil < 0 || bil > 99) { 
        printf("Anda Menginput Melebihi Limit Bilangan\n"); 
    } else if (bil == 0) { 
        printf("Nol\n"); 
    } else if (bil < 10) { 
        printf("Satuan\n"); 
    } else if (bil < 20) { 
        printf("Belasan\n"); 
    } else if (bil >= 10 && bil < 100) { 
        printf("Puluhan\n"); 
    } 
 
    return 0; 
} 

File: Modul 3 Sem 1/Phython/PRAK304-2410817110005-MuhammadRizqanKarim.py
bil = int(input("Masukkan bilangan (0-99): ")) 
if bil < 0 or bil > 99: 
    print("Anda Menginput Melebihi Limit Bilangan") 
elif bil == 0: 
    print("Nol") 
elif bil < 10: 
    print("Satuan") 
elif 10 < bil < 20: 
    print("Belasan") 
elif 10 <= bil < 100:  
    print("Puluhan")